packagecontrol.py

```diff
@@ -159,6 +160,7 @@ def json_dump_package(package):
         "date": package.date.isoformat(),
         "description": package.description,
         "download_url": package.download_url,
+        "download_count": package.download_count,

```

Comment:
the `packages.json` does not need this information. The keypirinha plugin has no use for this value, therefore it can be omitted for efficiency reasons.